index.html

```diff
@@ -949,10 +957,20 @@ <h3>
         <ol>
           <li>Let |promise:Promise| be [=a new promise=].
           </li>
-          <li>If [=this=]'s [=relevant global object=]'s [=associated
-          Document=] is not [=allowed to use=] the [=policy-controlled
-          feature=] named `"hid"`, [=reject=] |promise| with a
-          "{{SecurityError}}" {{DOMException}} and return |promise|.
+          <li>Let |document:Document| be null</li>
+          <li>If [=this=]'s [=relevant global object=] is {{Window/window}},
```

Comment:
Let's tweak these checks so they are worded like the definitions in https://www.w3.org/TR/service-workers/#service-worker-client-concept

  If [=this=]'s [=global object=] is a {{Window}} object,

Reply:
Oh, that makes sense. Yes, let's use [=relevant global object=].

Reply:
Thx for confirming, done.